src/appleseed/renderer/kernel/shading/shadingpoint.cpp

```diff
@@ -1114,7 +1114,7 @@ void ShadingPoint::initialize_osl_shader_globals(
 
         // Opaque state pointers.
         m_shader_globals.renderstate = const_cast<ShadingPoint*>(this);
-        memset(&m_osl_trace_data, 0, sizeof(OSLTraceData));
+        m_osl_trace_data = {};
```

Comment:
Warning of using `memset` with non trivial datatype. Value initialization should act as memset to 0 with primitive datatypes, so this should be equivalent as far as I can see.

Reply:
Agreed, that's better.